packages/core/src/App/Containers/RealAccountSignup/status-dialog-container.jsx

```diff
@@ -43,91 +48,62 @@ class StatusDialogContainer extends React.Component {
-                    })}
-                >
-                    <MainIcon currency={currency} />
-                    {this.status === EXPERIAN.SUCCESS && <Checkmark className='bottom-right-overlay' />}
-                    {this.status === EXPERIAN.WARN && <Warning className='bottom-right-overlay' />}
-                    {this.status === EXPERIAN.DANGER && <Danger className='bottom-right-overlay' />}
-                </div>
-                <div
-                    className={classNames('status-dialog__body-area', {
-                        'status-dialog__body-area--no-grow': isMobile(),
-                    })}
-                >
-                    <DialogHeading status={this.status} landing_company_shortcode={landing_company_shortcode} />
-                    <DialogMessage
-                        status={this.status}
-                        landing_company_shortcode={landing_company_shortcode}
-                        currency={currency}
-                        is_isle_of_man_residence={is_isle_of_man_residence}
-                        is_belgium_residence={is_belgium_residence}
-                    />
-                </div>
-                <DialogButtons
+    return (
+        <Div100vhContainer className='status-dialog' is_disabled={isDesktop()} height_offset='40px'>
+            <div
+                className={classNames('status-dialog__icon-area', {
+                    'status-dialog__icon-area--large': icon_size === 'large',
+                    'status-dialog__icon-area--xlarge': icon_size === 'xlarge',
+                })}
+            >
+                <MainIcon currency={currency} />
+                {getStatus() === EXPERIAN.SUCCESS && <Checkmark className='bottom-right-overlay' />}
+                {getStatus() === EXPERIAN.WARN && <Warning className='bottom-right-overlay' />}
+                {getStatus() === EXPERIAN.DANGER && <Danger className='bottom-right-overlay' />}
+            </div>
+            <div
+                className={classNames('status-dialog__body-area', {
+                    'status-dialog__body-area--no-grow': isMobile(),
+                })}
+            >
+                <DialogHeading status={getStatus()} landing_company_shortcode={landing_company_shortcode} />
+                <DialogMessage
+                    status={getStatus()}
                     landing_company_shortcode={landing_company_shortcode}
-                    status={this.status}
-                    closeModal={closeModal}
-                    closeModalAndOpenPOI={this.closeModalAndOpenPOI}
-                    closeModalAndOpenPOA={this.closeModalAndOpenPOA}
-                    closeModalAndOpenCashier={this.closeModalAndOpenCashier}
-                    switchToVirtual={switchToVirtual}
+                    currency={currency}
+                    is_isle_of_man_residence={is_isle_of_man_residence}
+                    is_belgium_residence={is_belgium_residence}
```

Comment:
These properties are not used in `DialogMessage` component. Can we remove these?